src/components/Homepage/HomePage.js

```diff
@@ -0,0 +1,15 @@
+import Banner from './Banner'
+import Feed from './Feed'
+import Article from './Article'
+
+
+function HomePage(){
+  return(
+    <div>
+      <Banner/>
+      <Feed/>
```

Comment:
Popular tags component should live along side feed and Banner on the top level of the home page.

Reply:
I moved "Feed" component into "Banner" component. To have "Banner" on the top level of the home page.